app/repos/relationships/person.js

```diff
@@ -1,42 +1,48 @@
-const personRelationship = (sequelize) => [
-  {
-    model: sequelize.models.person_address,
-    as: 'addresses',
-    include: [
-      {
-        model: sequelize.models.address,
-        as: 'address',
-        include: [
-          {
-            attribute: ['country'],
-            model: sequelize.models.country,
-            as: 'country'
-          }
-        ]
-      }
-    ]
-  },
-  {
-    model: sequelize.models.person_contact,
-    as: 'person_contacts',
-    separate: true, // workaround to prevent 'contact_type_id' being truncated to 'contact_type_i'
-    include: [
-      {
-        model: sequelize.models.contact,
-        as: 'contact',
-        include: [
-          {
-            model: sequelize.models.contact_type,
-            as: 'contact_type'
-          }
-        ]
-      }
-    ]
+const { Op } = require('sequelize')
+
+const personRelationship = (sequelize) => [{
```

Comment:
I need to check the generated SQL here as not 100% confident this is correct. Also, it affect all other queries that use this relationship so might have other impacts. I'll try to assess the generated SQL later today

Reply:
address history is returned by the repo, but is filtered out by the dto mapper.